scripts/quickstart

```diff
@@ -68,8 +89,16 @@ if [ "$show_help" = "true" ]; then
   exit 1
 fi
 
+if [ ! -z "$ACME_CERT_ENABLED" ]; then
+  echo "${BLUE}[INFO]${RESET} ACME Certificate request is ${BOLD}ENABLED${RESET}."
+  
+  if ! domainResolves "api.${DOMAIN}"; then
+    echo "${YELLOW}[WARN]${RESET} Unable to resolve \"api.${DOMAIN}\"! This might mean that you cannot use an ACME issued certificate."
```

Comment:
Can you break this message in two lines?

Reply:
Sure thing man